### ISSUE TITLE
It is not possible to use GPUs or NPUs on Android

### ISSUE THREAD

**Issue by ZTMIDGO** [freedomtan/keras_cv_stable_diffusion_to_tflite#3]
I tried these models:
sd_diffusion_model_dynamic.tflite
sd_diffusion_model_first.tflite
sd_diffusion_model_first_fixed_batch.tflite
sd_diffusion_model_second.tflite
sd_diffusion_model_second_fixed_batch.tflite

But none of them can be accelerated using GPUs or NPUs

**Comment by freedomtan** [comment]
> I tried these models: sd_diffusion_model_dynamic.tflite sd_diffusion_model_first.tflite sd_diffusion_model_first_fixed_batch.tflite sd_diffusion_model_second.tflite sd_diffusion_model_second_fixed_batch.tflite
> 
> But none of them can be accelerated using GPUs or NPUs

Well, YES and NO. NO, they are not NNAPI or say NNAPI delegate compatible mainly because of group normalization. It could be fixed by some techniques such as converting group normalization to layer normalization. I didn't try GPU delegate I guess it's similar.

**Comment by fuerpy** [comment]
https://github.com/tensorflow/tensorflow/pull/60794
hi guys，I saw this pull request ，think will be supported in some days.

**Comment by ZTMIDGO** [comment]
@fuerpy Perfect, I have finished running sd on Android with onnxruntime, unfortunately the accelerator cannot be used, if TensorFlow will support accelerator soon, I will fully develop the sd running TensorFlow version on Android

https://github.com/ZTMIDGO/Android-Stable-diffusion-ONNX

**Comment by freedomtan** [comment]
for your reference, https://www.slideshare.net/kstan2/stable-diffusion-on-android-with-nnapi
